chia/rpc/full_node_rpc_api.py

```diff
@@ -470,7 +470,11 @@ async def get_network_space(self, request: Dict) -> Optional[Dict]:
 
         newer_block = await self.service.block_store.get_block_record(newer_block_bytes)
         if newer_block is None:
-            raise ValueError(f"Newer block {newer_block_hex} not found")
+            # It's possible that the peak block has not yet been committed to the DB, so as a fallback, check memory
+            try:
+                newer_block = self.service.blockchain.block_record(newer_block_bytes)
```

Comment:
@mariano54 is this a more expensive call? does it need to lock the blockchain mutex? I assume there's some reason to not reach for this call in the first place